components/gantry/oneaxis/oneaxis.go

```diff
@@ -247,9 +210,8 @@ func (g *oneAxis) homeTwoLimSwitch(ctx context.Context) error {
 	g.positionLimits = []float64{positionA, positionB}
 
 	// Go backwards so limit stops are not hit.
-	x := g.linearToRotational(0.8 * g.lengthMm)
-	err = g.motor.GoTo(ctx, g.rpm, x, nil)
-	if err != nil {
+	x := g.rotationalToLinear(0.8 * g.lengthMm)
```

Comment:
:eyes: Oof, this looks like a very frustrating bug to find! 

...actually are you sure about this? The variables in the argument look to me like they're linear because they're in millimeters. I can't tell what the units on the return value (`x`) are, so I'm unsure whether you've just fixed or introduced the bug.